src/SFML/Audio/SoundSource.cpp

```diff
@@ -189,22 +383,12 @@ SoundSource& SoundSource::operator=(const SoundSource& right)
 ////////////////////////////////////////////////////////////
 SoundSource::Status SoundSource::getStatus() const
 {
-    ALint status;
-    alCheck(alGetSourcei(m_source, AL_SOURCE_STATE, &status));
-
-    switch (status)
-    {
-        case AL_INITIAL:
-        case AL_STOPPED:
-            return Stopped;
-        case AL_PAUSED:
-            return Paused;
-        case AL_PLAYING:
-            return Playing;
-    }
+    if (const auto* sound = static_cast<const ma_sound*>(getSound()); sound && (ma_sound_is_playing(sound) == MA_TRUE))
+        return Playing;
 
     return Stopped;
```

Comment:
Issue here: `Status::Paused` is never returned.

Reply:
Fixed.